### ISSUE TITLE
add base24 support?

### ISSUE THREAD

**Issue by KiaraGrouwstra** [Misterio77/nix-colors#42]
`base24` [adds bright colors for use in terminals](https://github.com/tinted-theming/base24/blob/master/styling.md) and seems supported by [various terminals](https://github.com/tinted-theming/base24#supported-applications-base24). it might be cool to see these supported here as well. there are [a couple such themes](https://github.com/tinted-theming/base24/#scheme-repositories) currently, whereas such schemes may be converted back to base16 using a simple [fallback table](https://github.com/tinted-theming/base24/blob/master/styling.md#colours).

**Comment by MobSenpai** [comment]
Yeah i would also love to have more colors for theming instead of limited to 16 colors. Can i use/call more colors with this atm? If I use a custom colorscheme with more than 16 colors and names other than base**. @Misterio77

**Comment by Misterio77** [comment]
Hey all!

Yeah, the module already supports arbitrary color names. The only thing hardcoded to consider 00-0F is the [`schemeFromYAML`](https://github.com/Misterio77/nix-colors/blob/main/lib/core/schemeFromYAML.nix#L61) function.

I plan on updating that function so that we can work with [Tinted Theming's new common format](https://github.com/tinted-theming/home/blob/main/builder.md#schemes), while keeping compatibility with the older base16/24 format (with the colors at top level). That new format is very similar to what I've been doing here in nix-colors, so I'll probably also rename our options (keeping backwards compatibility, of course) to match.
